datahub-web-react/src/app/shared/recommendation.tsx

```diff
@@ -0,0 +1,44 @@
+import { useGetSearchResultsForMultipleQuery } from '../../graphql/search.generated';
+import { EntityType } from '../../types.generated';
+
+export function GetTagRecommendation() {
+    const { data } = useGetSearchResultsForMultipleQuery({
+        variables: {
+            input: {
+                types: [EntityType.Tag],
+                query: '*',
+                start: 0,
+                count: 5,
+            },
+        },
+    });
+    return data?.searchAcrossEntities?.searchResults;
+}
+
+export function GetOwnerRecommendation() {
+    const { data } = useGetSearchResultsForMultipleQuery({
+        variables: {
+            input: {
+                types: [EntityType.CorpGroup, EntityType.CorpUser],
+                query: '*',
+                start: 0,
+                count: 5,
+            },
+        },
+    });
+    return data?.searchAcrossEntities?.searchResults;
```

Comment:
For each of these methods, let's return Array<Entity>

That means that we can do the following:

```
return data?.searchAcrossEntities?.searchResults?.map(searchResult => searchResult.entity) || []
```

Reply:
Checked and fixed.